src/xenialdan/Backpack/BackpackInventory.php

```diff
@@ -22,9 +23,20 @@ public function getVaultData(): Backpack
 		return $this->vault_data;
 	}
 
-	public function getName(): string
```

Comment:
So no more custom names? rip

Reply:
They work but Inventory::getName() has been unused for a long time, InvMenu::setName still works (https://github.com/Muqsit/Backpack/blob/85575cb0be23e9231f27b936f71e56ffbd676f68/src/xenialdan/Backpack/Backpack.php#L52)